Fluid/TemplateOptions.cs

```diff
@@ -73,6 +73,14 @@ public class TemplateOptions
         /// </summary>
         public bool Greedy { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets the <see cref="StringComparer"/> used for matching keys of scope values.
+        /// </summary>
+        public StringComparer StringComparer
```

Comment:
Where this property is used?

Reply:
At my unit test to change the `StringComparer` all way down to the scope.

Reply:
Wait, my test is failing because the property was removed.

Reply:
Can I fix this?

Reply:
> Can I fix this?
Yes

Reply:
I will check ..